R/measures.R

```diff
@@ -47,16 +49,26 @@
 #'
 #' @examples
 #' \dontrun{
-#' riem_measures(station = "VOHY", date_start = "2000-01-01", date_end = "2016-04-22")
+#' riem_measures(station = "VOHY", date_start = "2000-01-01", date_end = "2016-04-22", trace = 1e-4, missing = "empty")
 #' }
 riem_measures <- function(station = "VOHY",
                           date_start = "2014-01-01",
-                          date_end = as.character(Sys.Date())){
+                          date_end = as.character(Sys.Date()),
+                          trace = .0001,
+                          missing = "empty"){
 
-  base_link <- "https://mesonet.agron.iastate.edu/cgi-bin/request/asos.py/"
 
+  base_link <- "https://mesonet.agron.iastate.edu/cgi-bin/request/asos.py/"
 
+  if(!(trace %in% list("T", "null", "empty", .0001))) {
+    stop(call. = FALSE,
+         "trace must be one of 'T', 'null', 'empty' or 0.0001.")  # nolint
+  }
 
+  if(!(missing %in% list("M", "empty","null"))) {
```

Comment:
```suggestion
  if(!(missing %in% c("M", "empty","null"))) {
```

That one can be `c()`?